lib/lambda_ethereum_consensus/beacon_chain/state_transition.ex

```diff
@@ -0,0 +1,56 @@
+defmodule BeaconChain.StateTransition do
+  @moduledoc """
+  Core state_transition functions
+  """
+
+  @doc """
+  State_transition function managing the processing / validation of the `ExecutionPayload`
+  """
+  def process_execution_payload(state, payload, config) do
```

Comment:
We should not pass a config object to every function, see this issue: https://github.com/lambdaclass/lambda_ethereum_consensus/issues/243